restapi/api.go

```diff
@@ -21,10 +25,31 @@ func Router(db *gorm.DB) (http.Handler, error) {
 	}
 
 	rest.GetHandler(r, "/health", api.health)
-
+	rest.PostHandler(r, "/authors", api.authorCreate)
 	return r, nil
 }
 
+func (api api) authorCreate(w http.ResponseWriter, r *http.Request) render.Renderer {
+	// Declare a new Story struct.
+	var s []models.Author
+
+	// Try to decode the request body into the struct. If there is an error,
+	// respond to the client with the error message and a 400 status code.
+	err := json.NewDecoder(r.Body).Decode(&s)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return rest.MsgStatusOK("Error")
+	}
+
+	if err := api.database.Create(&s).Error; err != nil {
+		return rest.ErrInternal(zap.S(), err)
```

Comment:
after rebasing on master, `api` has a field called `logger` so now you can just do `api.logger`